client/src/create.js

```diff
@@ -36,20 +43,166 @@ class Create extends React.Component {
       name: inputValue('name').toLowerCase(),
     })
 
-    console.log(json)
-
     this.setState({
       containerName: json.containerInfo.name,
-      redirect: true,
+      redirectContainer: true,
+    })
+  }
+
+  async postSite(event) {
+    event.preventDefault()
+    event.nativeEvent.stopImmediatePropagation()
+    const doc = shadowDocument()
+    const inputValue = name => doc.getElementById(name).value
+
+    const json = await apiCall('POST', '/instances/new', {
+      url: inputValue('url'),
+      name: inputValue('name').toLowerCase(),
+      type: 'site',
     })
+    window.location.replace(`//${json.containerInfo.subdomain}.localhost:8080`)
```

Comment:
fix link

Reply:
You can use `window.location` for the domain